### PR TITLE
Feature/postgre database tests

### PR DESCRIPTION
**Pull request by HeiligeMeier**

### CODE REVIEW

**Review by ThmsRchhrzr — APPROVED** [review]
wenn alle Tests mit der Postgre funktionieren, dann OK

File: backend/codingchallenge/api/tests/views/application/test_editApplication.py [hunk 1]
```diff
@@ -28,10 +28,10 @@ def setUp(self):
 
         from django.contrib.auth.models import User
 
-        print(Application.objects.count())
-        print(User.objects.count())
-        print(User.objects.first())
-        print(getattr(User.objects.first(), "id"))
+        # print(Application.objects.count())
```
Comment (Marcodaum) [inline backend/codingchallenge/api/tests/views/application/test_editApplication.py:31]: remove print statements completely

File: backend/codingchallenge/codingchallenge/settings.py [hunk 2]
```diff
@@ -98,6 +100,24 @@
     }
 }
 
+# if 'test' in sys.argv:
```
Comment (Marcodaum) [inline backend/codingchallenge/codingchallenge/settings.py:103]: remove commented code

**Review by Marcodaum — CHANGES_REQUESTED** [review]

**Review by lorenzschreyer — APPROVED** [review]

**Review by FabianBretz — CHANGES_REQUESTED** [review]
Is this related to #252 ?
If so, please link this issue to the PR, so that it gets automatically closed.

**Review by FabianBretz — APPROVED** [review]

**Review by Marcodaum — APPROVED** [review]

**Pull request merged**